xml/storage_filesystems.xml

```diff
@@ -2017,12 +2017,43 @@ Label: none uuid: 40123456-cb2c-4678-8b3d-d014d1c78c78
      file systems. Do <emphasis>not</emphasis> use <command>wiper.sh</command>
      on the Btrfs file system as it may damage your data. Instead, use
      <command>/usr/share/btrfsmaintenance/btrfs-trim.sh</command> which is part
-     of the <systemitem class="resource" >btrfsmaintenance</systemitem>
-     package.
+     of the <package>btrfsmaintenance</package> package.
     </para>
    </warning>
   </sect2>
 
+  <sect2 xml:id="sec-filesystems-trouble-balance">
+   <title>Btrfs: Balancing Data across Devices</title>
+   <para>
+    The <command>btrfs balance</command> command is part of the
+    <package>btrfsmaintenance</package> package. It balances block groups on a
+    Btrfs file systems. For example, if you have 1 TB drive with 600 GB used by
+    data and you add another 1 TB drive, the balancing will theoretically
+    result in having 300 GB used space on each drive.
```

Comment:
This is a good example. A better example will be of usage, that is compacting half empty block groups based on usage. That will justify why it needs to be run weekly.